### ISSUE TITLE
Exporting Diagram gives empty SVG file

### ISSUE THREAD

**Issue by TooBusyPT** [QuantStack/jupyterlab-drawio#106]
When I try to export a diagram to a SVG file the result is an empty SVG file with the following name "untitled.svg" even if I have my diagram saved with a file name that I specified.

**Comment by Kevinkevin189** [comment]
exact same thing occurred to me

**Comment by davidjgraph** [comment]
To vote, thumbs up the original issue. On Github writing "+1" or "me too" is discouraged.

**Comment by andreaconti** [comment]
yeah, me too with the latest version

**Comment by Ekkehard** [comment]
Yup - still an open issue - found exactly the same behavior in the version I installed yesterday.  It's a bummer since how else would one incorporate the newly created graphic in a notebook without leaving jupyter-lab...

**Comment by weifpeng** [comment]
same

**Comment by pengzhenjia** [comment]
me too
